openedx/features/edly/middleware.py

```diff
@@ -0,0 +1,24 @@
+"""
+Edly Organization Access Middleware
+"""
+from logging import getLogger
+from django.http import Http404
+
+from openedx.features.edly.utils import user_has_edly_organization_access
+
+logger = getLogger(__name__)
+
+
+class EdlyOrganizationAccessMiddleware(object):
+    """
+    Django middleware to validate edly user organization access based on request.
+    """
+
+    def process_request(self, request):
+        """
+        Validate edly user organization access based on request.
```

Comment:
`Validate logged in user's access based on request site and its linked edly sub organization.`